src/plugins/relay/relay-network.c

```diff
@@ -85,13 +86,22 @@ relay_network_set_ssl_cert_key (int verbose)
                 if (verbose)
                 {
                     weechat_printf (NULL,
-                                    _("%s%s: warning: no SSL certificate/key "
-                                      "found (option relay.network.ssl_cert_key)"),
-                                    weechat_prefix ("error"), RELAY_PLUGIN_NAME);
+                                    _("%s%s: GnuTLS error: %s: %s"),
+                                    weechat_prefix ("error"), RELAY_PLUGIN_NAME,
+                                    gnutls_strerror_name(ret),
+                                    gnutls_strerror(ret));
                 }
             }
             free (certkey_path2);
```

Comment:
In the else clause, this variable is not freed.
You should move this block after the else block and replace it with:

```C
    if (certkey_path2)
        free (certkey_path2);
```